packages/guides/src/Nodes/Metadata/AuthorsNode.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides\Nodes\Metadata;
+
+use phpDocumentor\Guides\Nodes\ListNode;
+use phpDocumentor\Guides\Nodes\Node;
+use phpDocumentor\Guides\Nodes\ParagraphNode;
+use phpDocumentor\Guides\Nodes\RawNode;
+
+use function count;
+use function explode;
+
+/**
+ * The authors element is a container for author information for documents with multiple authors.
+ */
+class AuthorsNode extends MetadataNode
+{
+    /** @var AuthorNode[] */
+    private array $authorNodes;
+
+    /**
+     * @param Node[] $children
+     */
+    public function __construct(string $plaintext, array $children)
+    {
+        $this->authorNodes = [];
+        if (count($children) === 1) {
```

Comment:
If we introduce factories for the nodes, it would be nice to move this logic onto the factory. As this might only apply to RST?

Reply:
Should this be a factory or rather a production?